amundsen_application/static/js/ducks/bookmark/reducer.ts

```diff
@@ -46,6 +68,12 @@ export function getBookmarksForUser(userId: string): GetBookmarksForUserRequest
     type: GetBookmarksForUser.REQUEST,
   }
 }
+export function getBookmarksForUserFailure(): GetBookmarksForUserResponse {
+  return { type: GetBookmarksForUser.FAILURE };
```

Comment:
Same as above for this response.

Reply:
We didn't use the value at all in the reducer so I deleted it. Seems like if we included it in the payload it should have been intentionally used in the reducer. Let me know what your intent was when writing this and I'll update accordingly.